### PR TITLE
integrate import module from most recent version of DataTools

### LINKED ISSUE #10: Integrate import module from `DataTools`
The integration of the import module from `DataTools` will need some slight modification of the module itself. The structure of zip-objects of `MapR` is very different to the structure of all the other apps zip-objects. I will extract the respective logic into a function, such that the specific logic for `MapR` can live in a separate function in the package `MapR`.

_Originally posted by @arunge in https://github.com/Pandora-IsoMemo/MapR/issues/1#issuecomment-1784722465_

### PR DESCRIPTION
**Pull request by arunge**
### New Features
- option to upload zip files via the import module from `DataTools`. Enables access to files from
  URL, Pandora Platform, local files, and online examples

image

image

### CODE REVIEW

**Comment by arunge** [comment]
@jan-abel-inwt @f-lukas I do not know why the checks are currently failing. Let us check this tomorrow.

**Comment by arunge** [comment]
@f-lukas you can start testing the feature locally.

**Comment by f-lukas** [comment]
> @jan-abel-inwt @f-lukas I do not know why the checks are currently failing. Let us check this tomorrow.

Looks like Version `23.09.6.2` is installed from the drat repo but `23.10` is required. What steps are needed to make DataTools `v23.10` available in drat?
From the job output: `DataTools    (NA -> 23.09.6.2   )`

File: R/01-mapPanelModule.R [hunk 1]
```diff
@@ -86,15 +83,23 @@ mapPanelServer <- function(id) {
...
-        if (substr(datapath, nchar(datapath) - 5 + 1, nchar(datapath)) != ".zipm") {
-          shinyjs::alert("Only .zipm files are supported!")
-        } else {
-          utils::unzip(datapath, exdir = tempdir())
-          image_list(convertJsonToDataFrame(file = paste0(tempdir(), "/image_list.json")))
-        }
-      }) %>% bindEvent(input[["file"]])
+        # reset
+        image_list(NULL)
+        # maybe reset the plot??
```
Comment (f-lukas) [inline R/01-mapPanelModule.R:96]: Good idea, thanks. I will do this in the next step.

**Review by f-lukas — APPROVED** [review]
Works :+1:  Thanks for implementing!

**Pull request merged**